### ISSUE TITLE
Bandwith limit based on schedule

### ISSUE THREAD

**Issue by zin105** [duplicati/duplicati#2921]
Would be great if you could set what times the bandwith limit option should be active. For example, If I'm doing a huge upload I would love if it could be limited in speed during the day, but go full throttle during the night when nothing else requires my bandwith.

**Comment by JonMikelV** [comment]
Do you think something like what's described in this post do what's needed?
https://forum.duplicati.com/t/remember-the-throttle-settings/1737/10

**Comment by ksignorini** [comment]
This is critical for a backup program. Please implement as soon as possible. If there's any way to vote this up, it's got my vote.

**Comment by sonofwatt** [comment]
Agreed. This is a critical feature. I don't want it saturating the upload link during business hours.

**Comment by JonMikelV** [comment]
A bounty has been created for this feature at https://www.bountysource.com/issues/52436819-feature-request-bandwith-limit-based-on-schedule

**Comment by warwickmm** [comment]
Issue #611 is slightly related in that it concerns the ability to _pause_ backups during certain time windows (e.g., no file scanning or an other resource-consuming tasks).

**Comment by fognar777** [comment]
I would love to see a feature like this, so I added $15 to the bounty.

**Comment by martlego** [comment]
I added 15$ to the bounty as well.

**Comment by likwidtek** [comment]
I would love this feature!!!  Especially on seeding a first backup to the cloud!

**Comment by martlego** [comment]
It appears that the bounty doesn't exist on Bountysource anymore. Were we able to transfer it anywhere else or is it gone?
